File: View_Bindings_java/app/src/main/java/com/example/view_bindings_java/MainActivity.java
package com.example.view_bindings_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    Button btn2;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn2 = findViewById(R.id.button2);
        btn2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(), "Button 2 is clicked", Toast.LENGTH_SHORT).show();
            }
        });
    }

    public void onButtonClicked(View v) {

        Toast.makeText(getApplicationContext(), "button is clicked", Toast.LENGTH_SHORT).show();
    }
}

File: View_Binding_Kotlin/app/src/main/java/com/example/view_binding_kotlin/MainActivity.kt
package com.example.view_binding_kotlin

import android.graphics.Color.blue
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.TextView
import android.widget.Toast
import androidx.core.widget.addTextChangedListener
import kotlinx.android.synthetic.main.activity_main.*;


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        with(textView)
        {
            text = "10";
            setTextColor(getColor(R.color.teal_700));
            textSize = 20f;

        }

        editText.apply()
        {
            addTextChangedListener{
                Log.i("ViewBinding",it.toString());
            }
        }

        button2.setOnClickListener(View.OnClickListener {
            Toast.makeText(it.context,"Button 2 is clicked !!",Toast.LENGTH_LONG).show();
        });


    }

    fun showToast(v: View)
    {
        Toast.makeText(v.context,"Button 1 is clicked!!",Toast.LENGTH_LONG).show();
    }


}